lib/grpc/connection.js

```diff
@@ -121,7 +121,7 @@ class GrpcConnection extends EventEmitter {
       // immediately.
       this._setState(connectionStates.connected, this.stream)
     } catch (err) {
-      logger.trace(
+      logger.warn(
         err,
         'Unexpected error establishing gRPC stream, will not attempt reconnect.'
       )
```

Comment:
Since there will be no reconnect... there's no reason to be stingy with the log level. This indicates a fundamental issue with the infinite tracing setup.